server/client.go

```diff
@@ -1419,7 +1445,7 @@ func (c *client) addShadowSubscriptions(acc *Account, sub *subscription) error {
 
 // canSubscribe determines if the client is authorized to subscribe to the
 // given subject. Assumes caller is holding lock.
-func (c *client) canSubscribe(subject []byte) bool {
```

Comment:
Any reason why you changed all subjects from []byte to string in functions checking for permission?

Reply:
Just reducing tons of string(x) and doing it one..